### PR TITLE
In app update changelog

### PR DESCRIPTION
**Pull request by annakocheshkova**
* [X] Has `CHANGELOG.md` been updated?

## Description

[AB#75315](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/75315)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,9 @@
 
 ## Version 3.0.1 (Under development)
 
-___
+### App Center Distribute
+
+* **[Feature]** Add `disableAutomaticCheckForUpdate` API that needs to be called before SDK start in order to turn off automatic check for update and a `checkForUpdate` API for a manual update check.
```
Comment (thyeggman) [inline CHANGELOG.md]:
```suggestion
* **[Feature]** Added a `DisableAutomaticCheckForUpdate` API that needs to be called before SDK start in order to turn off automatic check for update. 
* **[Feature]** Added a `CheckForUpdate` API to manually check for an update.
```

**Comment by owlstronaut** [comment]
Hmm, we are a bit inconsistent in how we present new apis. Sometimes we use the actual signature, sometimes we give an example of calling it. It doesn't look like we normally do this pseudocode version though

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,10 @@
 
 ## Version 3.0.1 (Under development)
 
-___
+### App Center Distribute
+
+* **[Feature]** Added a `DisableAutomaticCheckForUpdate` API that needs to be called before SDK start in order to turn off automatic check for update.
```
Comment (jaeklim) [inline CHANGELOG.md]: Somehow we are using present tense for CHANGELOG.

Comment (jaeklim) [reply CHANGELOG.md]:
```suggestion
* **[Feature]** Add a `disableAutomaticCheckForUpdate` API that needs to be called before SDK start in order to turn off automatic check for update. 
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,7 +2,10 @@
 
 ## Version 3.0.1 (Under development)
 
-___
+### App Center Distribute
+
+* **[Feature]** Added a `DisableAutomaticCheckForUpdate` API that needs to be called before SDK start in order to turn off automatic check for update. 
+* **[Feature]** Added a `CheckForUpdate` API to manually check for an update.
```
Comment (jaeklim) [inline CHANGELOG.md]:
```suggestion
* **[Feature]** Add a `checkForUpdate` API to manually check for update.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,7 +2,10 @@
 
 ## Version 3.0.1 (Under development)
```
Comment (guperrot) [inline CHANGELOG.md]:
```suggestion
## Version 3.1.0 (Under development)
```

**Pull request merged**